docs/guidance/EnterpriseScaleAnalytics-SelfService.md

```diff
@@ -0,0 +1,117 @@
+# Scaling and Self-Service in Enterprise-Scale Anayltics
+
+Efficient scaling within an Enterprise Data Platform is a much-desired goal of many organizations and IT organizations as business units should be enabled to build their own (data) solutions and applications that fulfill their requirements and needs. However. achieving this goal is a serious challenge as many existing data platforms are not built around the core concepts of scalability and decentralized ownership. This is not only true from an architectural standpoint, but also becomes noticeable when looking at the team structure within many existing data platforms.
+
+## Introduction
+
+Today, many enterprises have built large data platform monoliths around the concept of a single Azure Data Lake Gen2 account and potentially even a single storage container. In addition, a single Azure subscription is most often used for all data platform related tasks and the concept of subscription level scaling is absent in most architectural patterns. This can hinder continued adoption of Azure if users run into any of the [well-know Azure subscription or service-level limitations](https://docs.microsoft.com/en-us/azure/azure-resource-manager/management/azure-subscription-service-limits). Even though some of the constraints are soft limits, hitting these can still have a significant impact within a data platform that should be avoided at best.
+
+Also, many organizations have organized their data platform architecture and teams around functional responsibilities and pipeline stages that need to be applied to the various data assets of an organization. As a result, specialized teams within a data platform own an orthogonal solution such as ingestion, cleansing, aggregation or serving. This organizational and architectural concept leads to a dramatic loss of velocity, because when data consumers on the serving layer require new data assets to be onboarded or functional changes to be implemented for a specific data asset the following processes will need to be followed o safely roll out these changes:
```

Comment:
Some organizations compose their data platform architecture and teams around functional responsibilities and pipeline stages that must be applied to the various data assets of an organization. As a result, several specialized teams within a data platform own an orthogonal solution spread across several phases such as ingestion, data cleansing, aggregation and/or serving. This organizational and architectural concept leads to a loss of velocity because of dependency on several teams. For example, when a data consumer at the serving layer requires a new data asset(s) to be onboarded or functional changes to be implemented, then multiple steps must be executed in a set sequene. The steps may involve: